src/widgets/gauge.rs

```diff
@@ -365,7 +378,7 @@ impl<'a> Widget for LineGauge<'a> {
                 .set_symbol(self.line_set.horizontal)
                 .set_style(Style {
                     fg: self.gauge_style.fg,
-                    bg: None,
+                    bg: self.gauge_background_style.fg,
```

Comment:
I wanted to keep it backwards compatible so I decided to introduce a new field `gauge_background_style`. However I'll admit I'm a bit uneasy with this line as it might be confusing to set `bg` to `fg`. It's just in the context of `LineGauge` foreground is used to style filled area and background is used to style empty area.